src/System.Text.Json/tests/Serialization/CustomConverterTests.cs

```diff
@@ -31,5 +31,39 @@ public static void MultipleConvertersInObjectArray()
             string jsonElementString = jsonElement.ToString();
             Assert.Equal(expectedJson, jsonElementString);
         }
+
+        [Fact]
+        public static void OptionsArePassedToCreateConverter()
+        {
+            TestFactory factory = new TestFactory();
+            JsonSerializerOptions options = new JsonSerializerOptions { Converters = { factory } };
+            string json = JsonSerializer.Serialize("Test", options);
+            Assert.Equal(@"""Test""", json);
```

Comment:
How would this be different if the options weren't passed or respected?

Reply:
It isn't likely to fail, it was just a quick sanity check that we aren't horribly broken in some other way. The thing that matters is that the options were passed (which I check below).